docs/data_management/res/cheaha_project_directory.csv

```diff
@@ -0,0 +1,16 @@
+Responsibilities,PI,Manager,Members,Research Computing
+Move unused shared data to LTS/archive,Yes ✔️,Yes ✔️,Yes ✔️,
+Manage backup and recovery,Yes ✔️,Yes ✔️,Yes ✔️,
+Ensure proper usage of shared storage,Yes ✔️,Yes ✔️,Yes ✔️,
+Add/remove member to/from the project space,Yes ✔️,Yes ✔️,,Upon PI/Manager request
+Hardware capital expenses,beyond default quota,,,up to default quota
```

Comment:
Add a line beneath "Hardware capital expenses"

Data center hosting expenses (until end of vendor service contract):
  - Research Computing: "Yes"